lib/usb/greaseweazleusb.cc

```diff
@@ -302,8 +301,8 @@ class GreaseWeazleUsb : public USB
                 cmd.writer()
                     .write_8(CMD_READ_FLUX)
                     .write_8(cmd.size())
-                    .write_le32(0) //ticks default value (guessed)
-                    .write_le32(revolutions + (synced ? 1 : 0));
+                    .write_le32((readTime + (synced ? _revolutions : 0))/ _clock)
```

Comment:
_revolutions or revolutions?

Reply:
_revolutions (revolutions is only computed for V22)

Reply:
OK. Not worth changing, but for reference I use nanoseconds_t for those to make it a bit clearer.

Reply:
...also there's a .clang-format in the root directory.